src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -80,6 +79,7 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
   val storageReadOnly = Seq(StorageScopes.DEVSTORAGE_READ_ONLY)
   // the scope we want is not defined in CloudbillingScopes, so we hardcode it here
   val billingScope = Seq("https://www.googleapis.com/auth/cloud-billing")
+  val spreadsheetScopes = Seq(SheetsScopes.SPREADSHEETS)
```

Comment:
Of note: we previously asked for drive, drive.file, and spreadsheets scopes from the end user. We only need spreadsheet scope, since we no longer *create* a spreadsheet. We now require the spreadsheet to exist already and be shared with the service account. By requiring a pre-existing spreadsheet we only need spreadsheet scope.